modules/sandboxed-containers-deleting-kataconfig-web-console.adoc

```diff
@@ -21,3 +32,5 @@ Deleting the `KataConfig` custom resource (CR) removes and uninstalls the `kata`
 . Click the Operator to open it, and then select the *KataConfig* tab.
 . Click the *Options* menu  {kebab} for the `KataConfig` resource, and then select *Delete `KataConfig`*.
 . Click *Delete* in the confirmation window.
+
+Wait for the `kata` runtime and resources to uninstall and for the worker nodes to reboot before continuing to the next step.
```

Comment:
This still leaves the kataconfigs.kataconfiguration.openshift.io CRD behind. We should document how to remove it using the web console just like we already do with the CLI. Basically go to the [Administration]->[CustomResourceDefinitions] menu, find the `KataConfig` CRD and delete it.